axis_backend/apps/clients/repositories/client_repository.py

```
@@ -0,0 +1,181 @@
+            Q(phone__isnull=False) | Q(contact_phone__isnull=False)
+        )
+
+    # Advanced Search
+
+    def search_clients(
+        self,
+        name: Optional[str] = None,
+        email: Optional[str] = None,
+        status: Optional[str] = None,
+        industry_id: Optional[str] = None,
+        is_verified: Optional[bool] = None,
```

Ruff (0.14.5)

Boolean-typed positional argument in function definition

(FBT001)